src/Commands/AmazonProductWidgetCommands.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Drupal\amazon_product_widget\Commands;
+
+use Drupal\amazon_product_widget\ProductService;
+use Drupal\Core\Queue\QueueFactory;
+use Drupal\Core\Queue\QueueWorkerManagerInterface;
+use Drupal\Core\Queue\SuspendQueueException;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Class AmazonProductWidgetCommands.
+ *
+ * Provides custom drush commands for queueing and updating product
+ * information.
+ *
+ * @package Drupal\amazon_product_widget\Commands
+ */
+class AmazonProductWidgetCommands extends DrushCommands {
+
+  /**
+   * ProductService.
+   *
+   * @var \Drupal\amazon_product_widget\ProductService
+   */
+  protected $productService;
+
+  /**
+   * Queue.
+   *
+   * @var \Drupal\Core\Queue\QueueInterface
+   */
+  protected $queue;
+
+  /**
+   * QueueWorkerManagerInterface.
+   *
+   * @var \Drupal\Core\Queue\QueueWorkerManagerInterface
+   */
+  protected $queueWorker;
+
+  /**
+   * AmazonProductWidgetCommands constructor.
+   *
+   * @param \Drupal\amazon_product_widget\ProductService $productService
+   *   ProductService.
+   * @param \Drupal\Core\Queue\QueueFactory $queue
+   *   QueueFactory.
+   * @param \Drupal\Core\Queue\QueueWorkerManagerInterface $queueWorker
+   *   QueueWorkerManagerInterface.
+   */
+  public function __construct(ProductService $productService, QueueFactory $queue, QueueWorkerManagerInterface $queueWorker) {
+    parent::__construct();
+    $this->productService = $productService;
+    $this->queue = $queue;
+    $this->queueWorker = $queueWorker;
+  }
+
+  /**
+   * Queues all products for renewal.
+   *
+   * @command apw:queue-product-renewal
```

Comment:
can you also add a reset-all-renewals which just runs $this->productService->getProductStore()->resetAll()